t/shdict.t

```diff
@@ -999,3 +999,25 @@ number expected, got string
 --- no_error_log
 [alert]
 [crit]
+
+
```

Comment:
Please use the `reindex` script to format your tests. We need 3 blank lines here, for example.

Reply:
done